LDAR_Sim/src/scheduling/generic_schedule.py

```diff
@@ -65,7 +65,8 @@ def _set_survey_plans(
             # TODO flush out what survey planner needs as inputs for the constructor
             survey_freq: int = site._survey_frequencies[self._method]
             # TODO: make sure this is in the correct place....
-            if survey_freq is None:
+            deploy_meth: bool = site.check_site_deployable(self._method)
```

Comment:
Nit; Can we rename this to something like do_site_deployment? Worried deploy meth might be confused later to be the name of the method